openedx/core/djangoapps/programs/tests/test_views.py

```diff
@@ -0,0 +1,135 @@
+"""
+Tests for the Programs.
+"""
+
+from mock import patch
+from provider.oauth2.models import Client
+from provider.constants import CONFIDENTIAL
+from unittest import skipUnless
+
+from django.conf import settings
+from django.test import TestCase
+
+from openedx.core.djangoapps.programs.views import get_course_programs_for_dashboard
+from openedx.core.djangoapps.programs.tests.mixins import ProgramsApiConfigMixin
+from student.tests.factories import AnonymousUserFactory, UserFactory
+
+
+@skipUnless(settings.ROOT_URLCONF == 'lms.urls', 'Test only valid in lms')
+class TestGetXSeriesPrograms(ProgramsApiConfigMixin, TestCase):
```

Comment:
in the test methods in this TestCase that use mock_get, please add assertions that mock_get is actually being called...